R/piping-ini.R

```diff
@@ -347,20 +346,22 @@
     checkmate::assert_choice(append, choices = ini$name)
     appendClean <- which(ini$name == append)
   } else {
-    cli::cli_abort("'append' must be NULL, logical, numeric, or character")
+    stop("'append' must be NULL, logical, numeric, or character", call. = FALSE)
   }
 
   lhs <- as.character(expr[[2]])
   wLhs <- which(ini$name == lhs)
   if (length(wLhs) != 1) {
-    stop("Cannot find parameter '", lhs, "'", call.=FALSE)
+    stop("cannot find parameter '", lhs, "'", call.=FALSE)
   } else if (length(appendClean) != 1) {
-    stop("Cannot find parameter '", after, "'", call.=FALSE)
+    stop("cannot find parameter '", append, "'", call.=FALSE)
```

Comment:
@billdenney is my reading of this code correct, it should be append, right?

Reply:
That's right